### ISSUE TITLE
`master` branch hangovers

### ISSUE THREAD

**Issue by da5nsy** [datalad/datalad-osf#180]
**Describe the bug**
There are a few places in this repo that still point to the `master` branch (which I think has been renamed `main`).
I noticed this while browsing https://docs.datalad.org/projects/osf/en/latest/index.html - all the "edit" buttons still point to the master branch.

**Suggested action**
I could _guess_ which of [these](https://github.com/search?q=repo%3Adatalad%2Fdatalad-osf%20master&type=code) need updating, but it seems wise to leave it to someone who knows the repo better 😅

**Comment by adswa** [comment]
uhh, this is a very good catch, thank you! You're right that the repo's default branch was renamed (we're slow with doing this for all the older repos, but we're getting there...), and I, too, believe this is some not-yet-noticed fallout of this. But I think none of the occurrences of ``master`` that  you linked are to blame here (though some need fixing, I'll take care). I rather think the automatic configuration on readthedocs.org failed, and it pointed ``latest`` to ``master``, even though we configured ``main`` to be the default branch. I have updated the config, and after many hard-refreshs of my browser the Edit button took me to the ``main`` branch - phew, and thanks again!
